roles/repo_setup/tasks/configure.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: Set cifmw_repo_setup_dlrn_hash_tag from content provider
+  ansible.builtin.set_fact:
+    cifmw_repo_setup_dlrn_hash_tag: "{{ content_provider_dlrn_md5_hash }}"
+  when: content_provider_dlrn_md5_hash is defined
```

Comment:
I have the feeling this will be defined to "nowhere" - is this really what we want?